docs/source/tutorial/06-add-more-info.mdx

```diff
@@ -31,9 +31,9 @@ query LaunchList {
 
 When you recompile, if you look in `LaunchListQuery.kt`, you'll see a new nested type, `Mission`, with the two properties you requested.
 
-Any GraphQL field can take arguments like `missionPatch` above, and arguments can be of scalar or complex types. In this case, `SMALL` is an enum in the GraphQL schema. It can take a finite list of values. If you look at the Docs pane in GraphQL Playground, you will see that size is an argument of type `PatchSize` that can only take two values: `SMALL` and `LARGE`
+Any GraphQL field can take arguments like `missionPatch` above, and arguments can be of scalar or complex types. In this case, `SMALL` is an enum in the GraphQL schema. It can take a finite list of values. If you look at the Schema section in Sandbox, you can see a list of the enums. You can then click in to see that `PatchSize` that can only take two values: `SMALL` and `LARGE`
```

Comment:
> You can then click in to see that `PatchSize` that can only take two values: `SMALL` and `LARGE`

Not 100% sure but should it be "You can then click in to see that `PatchSize` can only take two values: `SMALL` and `LARGE`"

Reply:
that sounds that is great 😆